### ISSUE TITLE
algunas dudas y sugerencias sobre la web

### ISSUE THREAD

**Issue by Pakillo** [ecoinfAEET/ecoinfAEET.github.io#5]
@ajpelu hay alguna forma de ver los cambios en mi fork antes de hacer pull request? Para ver cómo quedaría la web antes de hacer el pull request.

Otra duda: hay alguna forma de suscribirse a notificaciones de actualizaciones (RSS...)? He visto por algún lado que sí pero no sé cómo hacerlo. Estaría bien explicarlo por algún lado.

Por último, me gusta la introducción al grupo que aparece siempre como pagina principal, pero me temo que pueda verse la web al final un poco estática, y que gran parte de la actividad quede semi-oculta. Por ejemplo, ahora el anuncio del simposio aparece como un post en el blog, pero creo que estaria bien que este tipo de cosas aparecieran también en la pagina principal. 
Podriamos tener dos columnas en la pagina ppal, una con la intro al grupo y otra con las noticias o eventos, para que la gente pueda clicar?? O es algo imposible con este layout?
En fin, creo que estaria bien darle alguna vuelta al tema de darle mas visibilidad a lo que hacemos (que al final serán casi siempre posts en el blog)

que pensais @ajpelu @SaraVarela @ibartomeus ?

**Comment by ajpelu** [comment]
Buenos días chicos. 

Primero, gracias @Pakillo por el post (no me has dejado tiempo, jeje). 

Por otro lado, intento responder algunas cosillas. 

- Para poder ver los cambios en la web antes de hacer pull request etc, es tan sencillo como irte al directorio donde tengas el repo y en la consola escribir `jekyll serve`. Esto te monta en local la web (una url de estilo: `http://127.0.0.1:4000/` y puedes ir viendo los cambios a medida que los vayas haciendo. Es necesario que tengas jekyll instalado en tu maquina. Mas info aquí https://jekyllrb.com/docs/installation/ 

- Lo de RSS he visto que se puede hacer instalando un plugin, o tambien usando la[ aproximación](https://help.github.com/articles/atom-rss-feeds-for-github-pages/) de Github. Déjame que le eche un ratejo y os cuento. Pero en principio no es difícil (creo)

Finalmente, 
quizá podemos dejar la página de inicio con el blog, y mantener el quienes somos en una página aparte, ¿que os parece? 

buen día

**Comment by Pakillo** [comment]
Genial @ajpelu, muchas gracias :)

Para lo del RSS, la 'aproximación' de GitHub me parece perfecta. Lo que sea más fácil

Y quizas sea buena idea que el blog aparezca en la pagina principal, y el quienes somos se enlace aparte. Yo voto que si... Que os parece @SaraVarela @ibartomeus ?

**Comment by ibartomeus** [comment]
Sí, blog (y si puede ser solo con el tag "portada") en la principal y el
quien somos a parte me parece genial!

Gracias!

On Fri, Jun 10, 2016 at 4:25 PM Francisco Rodriguez-Sanchez <
notifications@github.com> wrote:

> Genial @ajpelu <https://github.com/ajpelu>, muchas gracias :)
>
> Para lo del RSS, la 'aproximación' de GitHub me parece perfecta. Lo que
> sea más fácil
>
> Y quizas sea buena idea que el blog aparezca en la pagina principal, y el
> quienes somos se enlace aparte. Yo voto que si... Que os parece
> @SaraVarela <https://github.com/SaraVarela> @ibartomeus
> <https://github.com/ibartomeus> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ecoinfAEET/ecoinfAEET.github.io/issues/5#issuecomment-225196299>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB8IgipPQaHfzIclXDJUV3-r0Jt2-Vxwks5qKXPigaJpZM4IyMQG>
> .
>
-- 
-- 
Ignasi Bartomeus PhD
www.bartomeuslab.com
@ibartomeus
Skype: nachobartomeus

Dpto. Ecología Integrativa
Estación Biológica de Doñana (EBD-CSIC)
Avda. Américo Vespucio s/n
Isla de la Cartuja
41092, Sevilla (Spain)

**Comment by SaraVarela** [comment]
todo genial :-)

**Comment by ajpelu** [comment]
Jelous gente!! 

Acabo de implementar lo de RSS. Ya está en la barra lateral con un icono. He probado con un sindicador y funciona. 

Por otro lado, ya he puesto la página principal dinámica con las entradas a los blog. La antigua página la he guardado en la carpeta _draft (no la eliminéis please). Habría que re-escribir el quienes somos y combinarlo con el contenido que teníamos en la entrada. 
Una cosa que perdemos con esta aproximación es que a la gente no le ofrecemos de un primer vistazo las diferentes formas de unirse (lista de correo, etc etc). ¿Que opináis? 

Seguimos.

**Comment by ibartomeus** [comment]
Muy buen trabajo!!

Sí, veo lo que dices. Seria posible mantener en la principal un párrafo de
quien somos, y las entradas (idealmente solo las de etiqueta "highlight")
debajo?

Yo esta semana voy muy pillado, pero la siguiente me lo miro si quereis.

Nacho



On Fri, Jun 17, 2016 at 12:49 AM Antonio Jesus <notifications@github.com>
wrote:

> Jelous gente!!
>
> Acabo de implementar lo de RSS. Ya está en la barra lateral con un icono.
> He probado con un sindicador y funciona.
>
> Por otro lado, ya he puesto la página principal dinámica con las entradas
> a los blog. La antigua página la he guardado en la carpeta _draft (no la
> eliminéis please). Habría que re-escribir el quienes somos y combinarlo con
> el contenido que teníamos en la entrada.
> Una cosa que perdemos con esta aproximación es que a la gente no le
> ofrecemos de un primer vistazo las diferentes formas de unirse (lista de
> correo, etc etc). ¿Que opináis?
>
> Seguimos.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ecoinfAEET/ecoinfAEET.github.io/issues/5#issuecomment-226636054>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB8IgkglqODhT009n7U7LJtm-y55Re2Pks5qMdLtgaJpZM4IyMQG>
> .
>
-- 
-- 
Ignasi Bartomeus PhD
www.bartomeuslab.com
@ibartomeus
Skype: nachobartomeus

Dpto. Ecología Integrativa
Estación Biológica de Doñana (EBD-CSIC)
Avda. Américo Vespucio s/n
Isla de la Cartuja
41092, Sevilla (Spain)
